new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/BuildApplication.java

```diff
@@ -156,30 +156,31 @@ public static Path buildApplication(Path appSrcPath, Map<String, String> antPara
       Kubernetes.copyFileToPod(namespace, webLogicPod.getMetadata().getName(),
           null, zipFile, Paths.get("/u01", zipFile.getFileName().toString()));
     } catch (ApiException | IOException  ioex) {
-      logger.info(ioex.getMessage());
+      logger.info("Exception while copying file to pod {0}", ioex);
     }
     try {
       //copy the build script to /u01 location inside pod
       Kubernetes.copyFileToPod(namespace, webLogicPod.getMetadata().getName(),
           null, BUILD_SCRIPT_SOURCE_PATH, Paths.get("/u01", BUILD_SCRIPT));
     } catch (ApiException | IOException  ioex) {
-      logger.info(ioex.getMessage());
+      logger.info("Exception while copying file to pod {0}", ioex);
```

Comment:
Same issue as previous comment.  Also, the error message is exactly the same as the previous one.  It might be better to include the filename in the log message so you know which file is failing.

This logger format is incorrect. If you are using this method
  `public void info(String msg, Object... params)`
then the code should be 
`logger.info("Exception while copying file to pod {0}", ioex.getMessage())`
 or if you are using this method
  `public void info(String msg, Throwable thrown)`
then the code should be 
`logger.info("Exception while copying file to pod", ioex)`